modules/ztp-du-bios-config-reference.adoc

```diff
@@ -89,3 +89,8 @@ The following table is a general recommendation for vDU cluster host BIOS config
 ====
 Enable global SR-IOV and VT-d settings in the BIOS for the host. These settings are relevant to bare-metal environments.
 ====
+
+[NOTE]
+====
+Enable both `P-states` and `C-State` to allow the use of power saving configurations through pod annotations.
```

Comment:
P-states should be "OS Controlled P-states". Also should probably be "...to allow per pod power management".